src/core/src/bootstrap/Constants.py

```diff
@@ -335,3 +337,11 @@ class UbuntuProClientSettings(EnumBackport):
         MAX_OS_MAJOR_VERSION_SUPPORTED = 18
         MINIMUM_CLIENT_VERSION = "27.14.4"
 
+    class StatusTruncationConfig(EnumBackport):
+        INTERNAL_FILE_SIZE_LIMIT_IN_BYTES = 126 * 1024
+        AGENT_STATUS_FILE_SIZE_LIMIT_IN_BYTES = 128 * 1024
+        MIN_TRUNCATED_PACKAGE_COUNT = 5
+        TRUNCATION_WARNING_MESSAGE = "Package lists were truncated to limit reporting data volume. In-VM logs contain complete lists."
+
+
```

Comment:
Why are these truncation constants being added in this PR? The PR is solely for adding a .complete.status file. There should not be any truncation related logic in here (that includes Constants). PRs should be completely independent on their own, adding truncation constants here adds dependency of the truncation PR on this one. And adds things that unrelated to the context of this PR.